guides/v3.7.0/testing/index.md

```diff
@@ -92,7 +92,7 @@ module('Component | counter', function(hooks) {
   test('it should count clicks', async function(assert) {
     this.set('value', 0);
 
-    await render(hbs`{{x-counter value=value onUpdate=( … )}}`);
+    await render(hbs`<XCounter @value={{this.value}} @onUpdate={{( … )}} />`);
```

Comment:
Do we still want to use `x-counter` here? considering we don't need it any more? i.e. we can have single word components like `<Counter>`

Reply:
This has already been converted.  Are you still seeing it?

Reply:
I'm still seeing `XCounter`